src/Mastermind/Mastermind.spec.ts

```diff
@@ -0,0 +1,9 @@
+import Mastermind from "./mastermind";
+
+describe("The Mastermind", function() {
+  test("should initialize a secret code", function() {
+    var mastermind = new Mastermind();
```

Comment:
Following my comment on the `Mastermind.ts` file, I would change this to be:

```suggestion
    var mastermind = new Mastermind([CodeColours.RED, CodeColours.BLUE, CodeColours.GREEN, CodeColours.YELLOW]);
```

Reply:
I emptied out the constructor so we can start off with a `red` test to fix up.